conans/client/cmd/copy.py

```diff
@@ -79,6 +79,13 @@ def package_copy(src_ref, user_channel, package_ids, cache, user_io, short_paths
         shutil.copytree(package_path_origin, package_path_dest, symlinks=True)
         user_io.out.info("Copied %s to %s" % (str(package_id), str(dest_ref)))
 
+    # The downloaded .tgz files can also be copied
+    src_download = src_layout.download_folder()
+    dst_download = dst_layout.download_folder()
+    rmdir(dst_download)
+    if os.path.isdir(src_download):
+        shutil.copytree(src_download, dst_download)
```

Comment:
We should only remove those listed in the `package_ids` list (they are already removed in the line 75) and copy only the files corresponding to those.